src/main/java/com/skankhunt220/api/transformer/CityTransformer.java

```diff
@@ -0,0 +1,12 @@
+package com.skankhunt220.api.transformer;
+
+import com.skankhunt220.entity.City;
+
+public class CityTransformer {
```

Comment:
Add to the project `CityDTO` and use CityDTO on controller and transform them to City using this transformer and back